tests/cpp/helpers.h

```diff
@@ -18,13 +18,22 @@
 #include <xgboost/objective.h>
 #include <xgboost/metric.h>
 #include <xgboost/predictor.h>
+#include <xgboost/generic_parameters.h>
+
+#include "../../src/common/common.h"
 
 #if defined(__CUDACC__)
 #define DeclareUnifiedTest(name) GPU ## name
 #else
 #define DeclareUnifiedTest(name) name
 #endif
 
+#if defined(__CUDACC__)
+#define NGPUS() 1
```

Comment:
Could you use `#define NGPUS 1`, without `()`, as there are no parameters?

Reply:
@trivialfis Second this comment.